### DISCUSSION TITLE
Update more often the go binaries?

### DISCUSSION THREAD

**Discussion by aborruso** [johnkerl/miller#405] [Q&A] [answered]
The go version of Miller will be really great.

To test it I can clone the code and compile it. But when I have tested it, I have had some problems. All it's ok using the binaries?

@johnkerl what could be way to have - in this developing phase - a quick and dirty go binary version of mlr, something like an "on the job" version?

If there's no quick and easy way, don't take the time. I will look forward to the next release.

Thank you

**Comment by johnkerl** [comment, marked as answer]
Yeah good call ...

* I totally get the desire for having them more often. It would benefit us all
* I don't know how to have Travis auto-build things in two diffrerent languages -- but https://stackoverflow.com/questions/27644586/how-to-set-up-travis-ci-with-multiple-languages might help ...

In any case, I do (on my laptop) have a build script which produces a binary, and a growing list of passing regression tests, and a shrinking list of failing ones, so it ought to be possible to automate this sooner than later ...

**Comment by johnkerl** [comment]
https://github.com/johnkerl/miller/issues/406
